.github/workflows/test_typing.yml

```diff
@@ -0,0 +1,25 @@
+name: Test typing
+
+on:
+  schedule:
+    # * is a special character in YAML so you have to quote this string
+    - cron:  '0 2 * * *'
+
+jobs:
+  typing:
+    name: Test typing on modules
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python 3.7
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.7
+      - name: Install napari
+        run: |
+          pip install -e .[all]
+          pip install -e .[testing]
+          pip install mypy
+      - name: Run mypy on typed modules
+        run: |
+          mypy napari/utils/settings
```

Comment:
I guess with time this will gradually grow!

Reply:
I will try to keep adding things slowly, one module per week or so, so we can slowly but steadily move in this direction :)